client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -98,7 +101,16 @@ export const useCommandsArrayWpcom = ( {
 
 		if ( ! sshUser ) {
 			removeLoadingNotice();
-			return navigate( `/hosting-config/${ siteSlug }` );
+			displayNotice(
+				__( 'SFTP/SSH credentials must be created before SSH connection string can be copied.' ),
```

Comment:
I was tempted to automatically create the SFTP/SSH credentials here but I refrained because I don't think it's necessarily expected behavior.